executor/insert_test.go

```diff
@@ -861,3 +861,190 @@ func (s *testSuite3) TestAutoIDIncrementAndOffset(c *C) {
 	c.Assert(err, NotNil)
 	c.Assert(err.Error(), Equals, "[autoid:8060]Invalid auto_increment settings: auto_increment_increment: 65536, auto_increment_offset: 65536, both of them must be in range [1..65535]")
 }
+<<<<<<< HEAD
+=======
```

Comment:
@Reminiscent 
Please fix the conflicts.

Reply:
@zimulala Update. PTAL